File: AlgorithmStudy/src/baekjoon_step3/Q11022.java
package baekjoon_step3;

import java.util.Scanner;

public class Q11022 {

	public static void main(String[] args)  {
		
		Scanner scanner = new Scanner(System.in);
		
		int num = scanner.nextInt();
		
		for(int i = 1; i < num+1; i++) {
			int a = scanner.nextInt();
			int b = scanner.nextInt();
			
			System.out.println("Case #" + i + ": " + a + " + " + b + " = " + (a+b));
		}
		
	}

}

File: src/main/kotlin/Q11022.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import java.util.*

fun main(args: Array<String>) = with(Scanner(System.`in`)){

    for(i in 1..nextInt()){
        val a = nextInt()
        val b = nextInt()

        println("Case $i: $a + $b = ${a+b}")
    }




}